lua/dial/map.lua

```diff
@@ -25,7 +25,7 @@ local function _cmd_sequence(direction, mode, group_name)
     -- command.select_augend_normal(vim.v.count, group_name)
     local setopfunc = cmdcr([[let &opfunc="dial#operator#]] .. direction .. "_" .. mode .. [["]])
     local textobj = util.if_expr(mode == "normal", cmdcr [[lua require("dial.command").textobj()]], "")
-    return select .. setopfunc .. "g@" .. textobj
+    return vim.v.count1..select .. setopfunc .. "g@" .. textobj
```

Comment:
In my environment it doesn't work, but by making the following changes, it works. How do you like it?

```suggestion
    return select .. setopfunc .. vim.v.count1 .. "g@" .. textobj
```

Reply:
I applied your suggestion, anything else I should do to get this merged?